demonstrations/tutorial_qubit_tapering.py

```diff
@@ -0,0 +1,295 @@
+r"""
+
+Qubit tapering
+==============
+
+.. meta::
+    :property="og:description": Learn how to taper off qubits
+    :property="og:image": https://pennylane.ai/qml/_images/ qubit_tapering.png
+
+.. related::
+    tutorial_quantum_chemistry Quantum chemistry with PennyLane
+    tutorial_vqe A brief overview of VQE
+    tutorial_givens_rotations Givens rotations for quantum chemistry
+    tutorial_adaptive_circuits Adaptive circuits for quantum chemistry
+    tutorial_differentiable_HF Differentiable Hartree-Fock
+
+
+*Author: PennyLane dev team. Posted: 17 May 2022. Last updated: 17 May 2022*
+
+The performance of variational quantum algorithms is considerably limited by the number of qubits
+required to represent wave functions. In the context of quantum chemistry, this
+limitation hinders the treatment of large molecules with algorithms such as the variational quantum
+eigensolver (VQE). Several approaches have been developed to reduce the qubit requirements for

```

Comment:
the VQE mention should be a hyperlink to the VQE tutorial :)
````suggestion
limitation hinders the treatment of large molecules with algorithms such as the :doc:`variational quantum
eigensolver (VQE) <tutorial_vqe>`. Several approaches have been developed to reduce the qubit requirements for
````